### PR TITLE
Gen styles

### PR DESCRIPTION
**Pull request by willyd332**
- Changed Hero Image
-  Changed Header (New format and now it appears/dissapears and is sticky)
- Widened view to take up full page
- Item modal grid restructure

### CODE REVIEW

File: src/components/item-view/data-section/ImpactRadar.js [hunk 1]
```diff
@@ -104,7 +107,6 @@ const ImpactDonut = ({ item }) => {
           />
         </VictoryChart>
       </div>
-  </div>
   );
 };
```
Comment (nellbhealy) [inline src/components/item-view/data-section/ImpactRadar.js:112]: Hangs off the bottom of the modal, but I imagine there's more work left to do on that anyway 👍

File: src/components/page-layout/Layout.js [hunk 2]
```diff
@@ -32,49 +30,61 @@ const createInitFoodList = () => {
...
-            Byar
-          </p>
-        </footer>
+            {React.cloneElement(children, { cartState, setCartState, toggleHeaderTop })}
+        </Grid>
+          <footer
+            className={styles.layoutFooter}
+          >
+            <p>
+              ©2020, A <a href="https://www.viget.com/">Viget</a> Project
```
Comment (nellbhealy) [inline src/components/page-layout/Layout.js]: Doug left a comment about this that we haven't addressed yet, just wanna make sure it's still on our radar

https://github.com/vigetlabs/EcoCart/pull/16#discussion_r452530486

File: src/components/page-layout/NavigationBar.js [hunk 3]
```diff
@@ -1,47 +1,83 @@
...
+import {
+  Button,
+  Grid,
+  Typography,
+} from '@material-ui/core';
 import { Link } from 'gatsby';
-import StorefrontIcon from '@material-ui/icons/Storefront';
 import ShoppingBasketIcon from '@material-ui/icons/ShoppingBasket';
-import HomeIcon from '@material-ui/icons/Home';
+import ecoLogo from '../../images/EcoCart_Logo.png';
```
Comment (nellbhealy) [inline src/components/page-layout/NavigationBar.js:11]: Not too important but this png is listed under the components section

File: src/components/page-layout/NavigationBar.js [hunk 4]
```diff
@@ -1,47 +1,83 @@
...
-      <Grid item>
-        <div>
+const NavigationBar = ({ toggleModal, resetHeaderTop }) => (
+    <Grid
+      container
+      justify="center"
+      alignItems="center"
+      className={styles.container}
+    >
+    <image src='public/EcoCart_Log.png' alt="Girl in a jacket" width="500" height="600" />
```
Comment (nellbhealy) [inline src/components/page-layout/NavigationBar.js]: Girl in a jacket?

File: src/components/page-layout/NavigationBar.js [hunk 5]
```diff
@@ -1,47 +1,83 @@
...
+          </Button>
+        </Link>
+      </Grid>
+      <Grid
+        item
+        sm={5}
+      >
+        <Link
+          to='/'
+          style={{ textDecoration: 'none' }}
```
Comment (nellbhealy) [inline src/components/page-layout/NavigationBar.js]: This is static and reused on line 59, so could be pulled out into the css module

File: src/components/page-layout/NavigationBar.js [hunk 6]
```diff
@@ -1,47 +1,83 @@
...
+        item
+        sm={5}
+        />
+      <Grid
+        item
+        sm={1}
+      >
         <Button className={styles.button} onClick={toggleModal}>
           <ShoppingBasketIcon className={styles.icon} />
         </Button>{' '}
```
Comment (nellbhealy) [inline src/components/page-layout/NavigationBar.js]: Is this string here on purpose?

**Review by nellbhealy — COMMENTED** [review]
Looks good, I left some comments but nothing crucial

**Review by nellbhealy — APPROVED** [review]

File: src/components/page-layout/NavigationBar.js (hunk 4 above)
Comment (willyd332) [reply src/components/page-layout/NavigationBar.js]: Whoops haha, copy/pasting RIP

**Pull request merged**